tests/baselines/reference/recursiveBaseConstructorCreation3.errors.txt

```diff
@@ -14,6 +14,6 @@ tests/cases/compiler/recursiveBaseConstructorCreation3.ts(10,18): error TS2339:
     }
     
     var bar = new xyz(); // Error: Invalid 'new' expression.

```

Comment:
Actually, I think I'll merge this the way it is. All the things I've tried so far to prevent cascading errors happen way before `resolveNewExpression` even gets to calling `getDefaultConstructSignatures`. So that change is going to be unrelated to and quite a bit more complicated than this fix.